src/sass/partials/_mq-mobile.scss

```diff
@@ -0,0 +1,92 @@
+@use "./variables" as *;
+
+@media (max-width: 480px) {
```

Comment:
since you're using sass, you can nest your @media within the style i.e.

h2 {
font-size: 12px;
@media (max-width: xxx) {
font-size: 10px;
}
}

just a thought! I just hate a separate media query partial lol

Reply:
I never thought about doing it that way! I like to have all of the media queries in their own partial just for ease of access, but I totally see why you'd prefer it this way!